cmd/console.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/kubefirst/kubefirst/configs"
+	"github.com/spf13/cobra"
+	"log"
+	"net/http"
+)
+
+var console = &cobra.Command{
+	Use:   "console",
+	Short: "starts ui server",
+	Long:  "starts app server for the Kubefirst console",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		config := configs.ReadConfig()
+		distFolder := fmt.Sprintf("%s/tools/console/dist", config.K1FolderPath)
+		fileServer := http.FileServer(http.Dir(distFolder))
+		http.Handle("/", fileServer)
+
+		fmt.Printf("Starting server at port 8080\n")
+	
+		if err := http.ListenAndServe(":8080", nil); err != nil {
```

Comment:
shall we use a less common port? it would avoid port conflict in case the user has some local service running in the most used http ports for local access.

many services make use of 8080 for running their service locally, we could have something like 8xxx where xxx is the port you choose

the http port could be a constant, so we dont need to repeat the value, or in case we need to change it, it a single point to change.